modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -0,0 +1,80 @@
+= Create API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can bundle a set of APIs to enable ease-of-use and efficiency for your API consumers. To create a bundle, you first create an API Group and then add the relevant API instances to it. If you want your API Group to be discovered by consumers, you must publish it to Anypoint Exchange.
+
+You can create API Groups using the following steps:
+
+*Step 1:* <<create-api-groups,Create an API Group>>
+
+The first step to creating an API Group that can be used by consumers, is create an API Group. 
+
+*Step 2:* <<adding-api-instances-to-groups,Add API instances to the new API Group>>
+
+After you create the API Group, you then add API instances to this group for the first time.
```

Comment:
It is true, but I think we need to clarify that in this flow you are creating a group, a group version, and a group instance. And the API instances are being added to the group instance, not the group

Reply:
Done! Thanks!